src/rt/array/concat.d

```diff
@@ -0,0 +1,83 @@
+module rt.array.concat;
+
+private extern (C) void[] _d_arraycatnTX(const TypeInfo ti, byte[][] arrs);
```

Comment:
Can this be imported instead of being declared as an `extern`?

Reply:
We should probably avoid that, IIRC druntime always uses extern declarations instead of imports for these internal functions. I guess it's to break cyclic module dependencies.